src/Integration/Telemetry/TelemetryTimer.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Timers;
+
+namespace SonarLint.VisualStudio.Integration
+{
+    public interface ITelemetryTimer : IDisposable
```

Comment:
Please move the `IDisposable` from the interface to the class to follow the pattern of other classes.

Reply:
Please also extract the interface into another file.